src/envoy/mixer/integration_test/auth_test.go

```diff
@@ -0,0 +1,158 @@
+  "mesh1.ip": "[1 1 1 1]",
+  "mesh2.ip": "[0 0 0 0 0 0 0 0 0 0 255 255 204 152 189 116]",
+  "mesh3.ip": "[0 1 0 0 0 0 0 0 0 0 0 0 0 0 0 8]",
+  "request.host": "localhost:27070",
+  "request.path": "/echo",
+  "request.time": "*",
+  "request.useragent": "Go-http-client/1.1",
+  "request.method": "GET",
+  "request.scheme": "http",
+  "source.uid": "POD11",
+  "source.namespace": "XYZ11",
+  "source.ip": "[127 0 0 1]",
+  "source.port": "*",
+  "target.name": "target-name",
+  "target.user": "target-user",
+  "target.uid": "POD222",
+  "target.namespace": "XYZ222",
+  "request.headers": {
+     ":method": "GET",
+     ":path": "/echo",
+     ":authority": "localhost:27070",
+     "x-forwarded-proto": "http",
+     "x-istio-attributes": "-",
+     "x-request-id": "*"
+  }
+}
+`
+
+const reportAttributes = `
+{
+  "context.protocol": "http",
+  "mesh1.ip": "[1 1 1 1]",
+  "mesh2.ip": "[0 0 0 0 0 0 0 0 0 0 255 255 204 152 189 116]",
+  "mesh3.ip": "[0 1 0 0 0 0 0 0 0 0 0 0 0 0 0 8]",
+  "request.host": "localhost:27070",
+  "request.path": "/echo",
+  "request.time": "*",
+  "request.useragent": "Go-http-client/1.1",
+  "request.method": "GET",
+  "request.scheme": "http",
+  "source.uid": "POD11",
+  "source.namespace": "XYZ11",
+  "source.ip": "[127 0 0 1]",
+  "source.port": "*",
+  "target.name": "target-name",
+  "target.user": "target-user",
+  "target.uid": "POD222",
+  "target.namespace": "XYZ222",
+  "request.headers": {
+     ":method": "GET",
+     ":path": "/echo",
+     ":authority": "localhost:27070",
+     "sec-istio-auth-userinfo": "eyJpc3MiOiI2Mjg2NDU3NDE4ODEtbm9hYml1MjNmNWE4bThvdmQ4dWN2Njk4bGo3OHZ2MGxAZGV2ZWxvcGVyLmdzZXJ2aWNlYWNjb3VudC5jb20iLCJzdWIiOiI2Mjg2NDU3NDE4ODEtbm9hYml1MjNmNWE4bThvdmQ4dWN2Njk4bGo3OHZ2MGxAZGV2ZWxvcGVyLmdzZXJ2aWNlYWNjb3VudC5jb20iLCJhdWQiOiJib29rc3RvcmUtZXNwLWVjaG8uY2xvdWRlbmRwb2ludHNhcGlzLmNvbSIsImlhdCI6MTUxMjc1NDIwNSwiZXhwIjo1MTEyNzU0MjA1fQ==",
+     "x-forwarded-proto": "http",
+     "x-istio-attributes": "-",
+     "x-request-id": "*"
+  },
+  "request.size": 0,
+  "response.time": "*",
+  "response.size": 0,
+  "response.duration": "*",
+  "response.code": 200,
+  "response.headers": {
+     "date": "*",
+     "content-type": "text/plain; charset=utf-8",
+     "content-length": "0",
+     ":status": "200",
+     "server": "envoy"
+  }
+}
+`
+
+func TestJWTAuth(t *testing.T) {
+	s := &TestSetup{
+		t:  t,
+		v2: GetDefaultV2Conf(),
+	}
+
+	// pubkey server is the same as backend server.
+	// Empty audiences.
+	AddJwtAuth(s.v2.HttpServerConf, &mccpb.JWT{
+		Issuer:              JWT_ISSUER,
+		JwksUri:             fmt.Sprintf("http://localhost:%d/pubkey", BackendPort),
+		JwksUriEnvoyCluster: JWT_CLUSTER,
+	})
+	if err := s.SetUp(); err != nil {
+		t.Fatalf("Failed to setup test: %v", err)
+	}
+	defer s.TearDown()
+
+	url := fmt.Sprintf("http://localhost:%d/echo", ClientProxyPort)
+
+	tag := "Good-Token"
+	headers := map[string]string{}
+	headers["Authorization"] = fmt.Sprintf("Bearer %v", longLiveToken)
```

Comment:
optional: this can be combined into a single declaration. 
```golang
headers := map[string]string{
    "Authorization": fmt.Sprintf("Bearer %v", longLiveToken),
}
```